cardinal/server/server.go

```diff
@@ -130,6 +131,58 @@ func getValueFromParams[T any](params interface{}, name string) (*T, bool) {
 	return value, true
 }
 
+func processTxParams(params interface{}, pathParam string, txNameToTx map[string]transaction.ITransaction, handler *Handler) (
```

Comment:
It can't be broken but does a struct really help here? The user either has to fill out a struct or fill out the parameters to a function. It's the same amount of filling out? Additionally not all the values are used in all cases. 

I can make it more readable by putting each parameter on a new line. I'll do that for now than we can discuss on tues.